HomeWork_03/DZ3.py

```diff
@@ -1,35 +1,32 @@
 #I love Python*42#
 
 # Zadanie 1
-x = ("I love Python")
+x = "I love Python"
 z = x*42
 print(z)
 
 #age_in_month#
 
 # Zadanie 2
-x = str("age_in_month")
-y = 37*12
-z = x+" "+str(y)
+x = month = 12
+y = age_in_year = 37
+z = age_in_month = x*y
```

Comment:
You can calculate age_in_month better by adding here the number of months passed after your birthday.
And you should use here not `x * y` but `age_in_years & month`